services/madoc-ts/src/frontend/site/pages/view-topic-type.tsx

```diff
@@ -21,15 +19,18 @@ export function ViewTopicType() {
         <TopicTypeHero />
       </Slot>
 
-      <ul>
-        {data?.topics.map(topic => (
-          <li key={topic.id}>
-            <HrefLink href={createLink({ topic: topic.slug })}>
-              <LocaleString>{topic.label}</LocaleString>
-            </HrefLink>
-          </li>
-        ))}
-      </ul>
+      <Slot name="topic-type-topics">
+        <TopicGrid />
+      </Slot>
+
+      <Slot name="topic-type-pagination">
+        <Pagination
+          page={data?.pagination ? data.pagination.page : 1}
+          totalPages={data?.pagination ? data.pagination.totalPages : 1}
+          stale={false}
+        />
```

Comment:
I think we might need a `<TopicTypePagination />` component that re-calls the data hook, similar to the Manifest/Collection paginations - so they can be overwritten in the page block.